src/parser.ts

```diff
@@ -178,7 +178,7 @@ export class Parser extends DiagnosticEmitter {
     var tn = new Tokenizer(source, this.diagnostics);
     tn.onComment = this.onComment;
     var statements = source.statements;
-    while (!tn.skip(Token.ENDOFFILE)) {
+    while (!tn.skip(Token.EOF)) {
```

Comment:
I would prefer if `Token.EOF` remained `Token.ENDOFFILE` to be in line with the other tokens that are also not abbreviations.